.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,30 +1,34 @@
 <!--- See what makes a good Pull Request at : https://github.com/hashicorp/terraform-provider-vault/blob/master/.github/CONTRIBUTING.md --->
 
-<!--- Please keep this note for the community --->
 
-### Community Note
+### Description
+<!--- Description of the change. For example: This PR updates ABC resource so that we can XYZ --->
 
-* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
-* Please do not leave "+1" comments, they generate extra noise for pull request followers and do not help prioritize the request
-
-<!--- Thank you for keeping this note for the community --->
 
 <!--- If your PR fully resolves and should automatically close the linked issue, use Closes. Otherwise, use Relates --->
 Relates OR Closes #0000
 
-Release note for [CHANGELOG](https://github.com/hashicorp/terraform-provider-vault/blob/master/CHANGELOG.md):
-<!--
-If change is not user facing, just write "NONE" in the release-note block below.
--->
 
-```release-note
+### Checklist
+- [ ] Added [CHANGELOG](https://github.com/hashicorp/terraform-provider-vault/blob/master/CHANGELOG.md) entry (only for user-facing changes)
+- [ ] Acceptance tests where run against all supported Vault Versions
```

Comment:
I am not exactly sure how to handle this. But it would be nice if we could link to the currently supported Vault Versions. Maybe once we get the TFVP docs updated with our support guidelines we can link to that?